spec/system/admin/searchgov_domain_management_spec.rb

```diff
@@ -1,21 +0,0 @@
-# frozen_string_literal: true
```

Comment:
please resurrect this file :)

Reply:
I am. It was a rename to searchgov_domain_management_spec.rb

Reply:
Resurrected.